tests/tablewalker.js

```diff
@@ -157,29 +157,29 @@ describe( 'TableWalker', () => {
 				{ row: 0, column: 0, index: 0, data: '00' },
 				{ row: 0, column: 1, index: 1, data: '01' },
 				{ row: 1, column: 0, index: 0, data: '10' },
-				{ row: 1, column: 1, index: 1, data: undefined }
+				{ row: 1, column: 1, index: 1, data: '01' }
 			], { includeSpanned: true } );
 		} );
 
-		it( 'should output spanned cells as empty cell', () => {
+		it( 'should output spanned cells', () => {
 			testWalker( [
 				[ { colspan: 2, rowspan: 3, contents: '00' }, '02' ],
 				[ '12' ],
 				[ '22' ],
 				[ '30', { colspan: 2, contents: '31' } ]
 			], [
 				{ row: 0, column: 0, index: 0, data: '00' },
-				{ row: 0, column: 1, index: 1, data: undefined },
```

Comment:
I see what you did with the walker and I now the docs are not in line with the `includedSpaned` intention.

The intention was to iterate either over existing table cells or by virtual (empty) places. For such empty places we can't output table cell as it will be in conflict with that use case.

The `include` spanned option was helpful to manage table modifications (extending table cells on a column, row, etc).

If you see the `cell` to be needed in the value we must indicate that current value is `spanned` by other cell.

However I can see that this is not used that much (this empty cell detection) and all the other features works o_0.

Anyway I think that we should keep this information - that a currently yielded table walker value belongs to a place spanned by other cell (so maybe `isSpanned: true` will be OK?